screens/Reports.js

```diff
@@ -115,7 +117,7 @@ export default class Reports extends Component<*, State> {
         <StatusBar barStyle="light-content" backgroundColor={OLD_GERANIUM} />
         <BackgroundGradient />
         <ChartsContainer
-          contentContainerStyle={{ alignItems: 'center', paddingVertical: 36 }}
+          contentContainerStyle={{ alignItems: 'center', paddingVertical: 48 }}
```

Comment:
I changed value https://github.com/kkemple/hwyd/blob/f46d25747ed57e921d3c49e51b43af2a2c597484/screens/Reports.js#L99 from `7` to `2` during dev so I could check out graph, which led to this `paddingTop` increase from `36` to `48`